pkg/driver/node_server.go

```diff
@@ -1715,7 +1715,14 @@ func (driver *Driver) NodeExpandVolume(ctx context.Context, request *csi.NodeExp
 		// figure out if volumePath is actually a staging path
 		stagedDevice, err := readStagedDeviceInfo(request.GetVolumePath())
 		if err != nil {
-			return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("Cannot get staging device info from volume path %s", err.Error()))
+			log.Tracef("Error retrieving stagedDevice from volume path %v",request.GetVolumePath())
+			log.Tracef(" Retrying with staged target path %v",request.GetStagingTargetPath())
+			// See if the "staging_target_path contains the deviceInfo.json
+			// This behaviour is peculiar to only k8s 1.19
+			stagedDevice, err = readStagedDeviceInfo(request.GetStagingTargetPath())
+			if err != nil {
+				return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("Cannot get staging device info from volume path %s", err.Error()))
```

Comment:
nit: "Cannot get staging device info from staging path %s"

I think we should have an error message which states that the device info cannot be retrieved either from "Volume path" or "Staging path".

Reply:
Done